### ISSUE TITLE
[Builder] [BUG] Tab will jump to first tab when component already on the form is dropped in

### ISSUE THREAD

**Issue by ShermanE1** [formio/formio.js#5309]
## Environment

- Hosting type: https://formio.github.io/formio.js/app/builder
- Browser: Chrome
- Version: 115.0.5790.110

## Steps to Reproduce

1. Add any Basic component to a form
2. Add a Tabs component with two tabs to the same form
3. Select the second tab
4. Drag & drop the component that was added in Step #1 to the second tab

## Expected behavior
Tabs component should still be in the second tab

## Observed behavior
Tabs component will switch to display the first tab

## Example
[screen-capture.webm](https://github.com/formio/formio.js/assets/115426742/57b8c9ce-6d97-45c5-825e-93aaa7154e47)

**Note:** A similar issue seems to have been fixed a while so that when dragging a new component from the side panel into a tab it renders the correct tab (#1689); however, the issue is still present when dragging a component that is already on the form into a tab.

**Comment by travist** [comment]
I just tried this out with the latest version of our renderer and it seems to be working for me. When I drag a component into the second tab it stays on the second tab. Can you provide a jsfiddle of this not working for you?

**Comment by ShermanE1** [comment]
Yes, this does appear to be fixed in the latest version. Thank you!

**Comment by mk160107** [comment]
After updating to the latest version, it seems that this error still exists. If you navigate to Tab 2 in [this jfiddle](https://jsfiddle.net/mk160107/5e8q9ptm/4/) and drag the Last Name field into it, the tab component will switch to the first tab. 

This only happens when dragging components that are already on the form into the second tab. If you drag a new component into the second tab, it will stay on the correct tab.
